packages/ember/lib/index.js

```diff
@@ -139,9 +138,6 @@ Ember.bind = metal.bind;
 Ember.Binding = metal.Binding;
 Ember.isGlobalPath = metal.isGlobalPath;
 
-if (EMBER_METAL_WEAKMAP) {
-  Ember.WeakMap = metal.WeakMap;
```

Comment:
Should this be put behind a deprecate method for the remainder of 2.X?

Reply:
Probably. @rwjblue?

Reply:
@thoov - The feature flag was never enabled, so it has never been published and usable by users.  AFAIK all users of this have to go through `ember-weakmap` which detects if `Ember.WeakMap` is set or else defines it...